src/lib/style/components/_cards.scss

```diff
@@ -214,6 +214,9 @@ $text-color-lighter: #898989;
   .SRC-cardMetadata {
     display: block;
     padding: $baseline-grid;
+    >* {
+      margin-left: 15px;
```

Comment:
The relevant change.  Children of SRC-cardMetadata (currently a single table element) should have a margin-left of 15px so that it aligns with card info above